Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -137,21 +145,67 @@ @implementation BPPacker
                     [BPUtils printInfo:INFO withString:@"Estimate not available for %@", test];
                 }
             }];
-            estimatedTimesByTestFilePath[xctFile.testBundlePath] = [NSNumber numberWithDouble:testBundleExecutionTime];
+            testEstimatedTimesByTestFilePath[xctFile.testBundlePath] = [NSNumber numberWithDouble:testBundleExecutionTime];
             totalTime += testBundleExecutionTime;
         }
     }
-    assert([estimatedTimesByTestFilePath count] == [xcTestFiles count]);
+    assert([testEstimatedTimesByTestFilePath count] == [xcTestFiles count]);
 
-    NSMutableArray<BPXCTestFile *> *bundles = [[NSMutableArray alloc] init];
+    [BPUtils printInfo:INFO withString:@"Stats before splitting..."];
+    NSMutableDictionary *inputDic = [NSMutableDictionary dictionary];
+    for(id key in testEstimatedTimesByTestFilePath) {
+        inputDic[[key substringFromIndex:[(NSString *)key rangeOfString:@"/" options:NSBackwardsSearch].location+1]] = [testEstimatedTimesByTestFilePath objectForKey:key];
+        NSLog(@"%@ = %@", [key substringFromIndex:[(NSString *)key rangeOfString:@"/" options:NSBackwardsSearch].location+1], [testEstimatedTimesByTestFilePath objectForKey:key]);
+    }
+
+    int minimumBundleTime = 180;
```

Comment:
This should be a constant

Reply:
Also, how was this number decided? Shouldn't we leave this up to the user?
Also "minimumBundleTime" seems a bit misleading

Reply:
Will rename and make it a constant.